test/check-application

```diff
@@ -2384,9 +2384,9 @@ class TestApplication(testlib.MachineCase):
         # Changing the port should allow creation of container
         b.set_input_text('#run-image-dialog-publish-0-host-port', '5001')
         b.click('.pf-v5-c-modal-box__footer #create-image-create-run-btn')
+        b.wait_not_present("#run-image-dialog-name")
+
         self.waitContainerRow(container_name)
-        container_sha = self.execute(False, f"podman inspect --format '{{{{.Id}}}}' {container_name}").strip()
-        self.waitContainer(container_sha, False, name=container_name, image=IMG_BUSYBOX, state='Running')
 
```

Comment:
Is that a rebase error, or did you deliberately drop this? Then the commit message should explain.

Reply:
Deliberate, this was there to make sure the dialog was closed but there is imo a better way :)